src/test/java/core/basesyntax/handlers/impl/RemoveOperationTest.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.handlers.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.handlers.FruitOperationHandler;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.OperationType;
+import core.basesyntax.model.dtos.FruitDtoTransaction;
+import org.junit.After;
+import org.junit.Test;
+
+public class RemoveOperationTest {
+    private FruitOperationHandler fruitOperationHandler = new RemoveOperation();
+
+    @After
+    public void afterEachTest() {
+        Storage.getFruits().clear();
+    }
+
+    @Test
+    public void addOperationWithCorrectData_Ok() {
```

Comment:
do we test the case when we want to remove more than we have?

Reply:
done